itou/metabase/management/commands/populate_metabase_matomo.py

```diff
@@ -26,12 +37,32 @@
     "zoom-prescripteurs": "tb 52 - Typologie de prescripteurs",
 }
 
+PRIVATE_DEPARTMENT_DASHBOARDS = {
+    "stats/cd/{}": "tb 118 - Données IAE CD",
+    "stats/ddets/hiring/{}": "tb 160 - Facilitation de l'embauche DREETS/DDETS",
+    "stats/ddets/iae/{}": "tb 117 - Données IAE DREETS/DDETS",
+    "stats/pe/conversion/main/{}": "tb 169 - Taux de transformation PE",
+    "stats/pe/delay/main/{}": "tb 168 - Délai d'entrée en IAE",
+    "stats/pe/state/main/{}": "tb 149 - Candidatures orientées PE",
+    "stats/pe/tension/{}": "tb 162 - Fiches de poste en tension PE",
+    "stats/siae/hiring/{}": "tb 165 - Recrutement SIAE",
+}
+
+PRIVATE_REGION_DASHBOARDS = {
+    "stats/dreets/hiring/{}": "tb 160 - Facilitation de l'embauche DREETS/DDETS",
+    "stats/dreets/iae/{}": "tb 117 - Données IAE DREETS/DDETS",
+    "stats/pe/conversion/main/{}": "tb 169 - Taux de transformation PE",
+    "stats/pe/delay/main/{}": "tb 168 - Délai d'entrée en IAE",
+    "stats/pe/state/main/{}": "tb 149 - Candidatures orientées PE",
+    "stats/pe/tension/{}": "tb 162 - Fiches de poste en tension PE",
```

Comment:
Pinaillage (tu peux ignorer) : ça fait un peu de duplication pas très DNRY avec les routes ([lien](https://github.com/betagouv/itou/blob/master/itou/www/stats/urls.py)). Peut-être que ca serait plus DNRY de déduire la string `"stats/pe/tension/{}"` depuis la view `views.stats_pe_tension` ? Par exemple un truc du genre `reverse('stats.view.stats_pe_tension')`, inspiration ici ([lien](https://stackoverflow.com/questions/10533302/how-to-get-the-url-path-of-a-view-function-in-django)).

Reply:
Tu as tout à fait raison, ce sera plus stable si j'utilise au moins en grande partie les reverse URLs.